src/app/addons/blog/data/posts/redwoodsdk-v1-beta.md

```diff
@@ -0,0 +1,144 @@
+
+We're excited to share that **RedwoodSDK v1.0.0 Beta** has officially
+landed! This marks an important step on our journey to a stable v1.0
+release, bringing greater stability, clarity, and performance to
+developers building full-stack React apps on Cloudflare Workers.
+
+👉 [Download the beta on
+GitHub](https://github.com/redwoodjs/sdk/releases/tag/v1.0.0-beta.2)\
+👉 [Get started with
+`create-rwsdk`](https://github.com/redwoodjs/create-rwsdk)\
+👉 [Migration guide for existing
+projects](https://docs.rwsdk.com/migrating)
+
+------------------------------------------------------------------------
+
+## Why v1 Matters
+
+Since the early days of RedwoodSDK, our mission has been simple:\
+**make it effortless to build full-stack React apps that run natively on
+the edge.**
+
+We've heard from developers who want three things:
+
+1.  **Predictability** --- A stable dev loop that works the same locally
+    as it does in production.\
+2.  **Clarity** --- Less hidden magic, more visibility into what your
+    code is doing.\
+3.  **Integration** --- First-class access to Cloudflare's primitives:
+    Durable Objects, D1, R2, and beyond.
+
+The **v1.0.0 Beta** is where those promises start to solidify.
+
+------------------------------------------------------------------------
+
+## What's New in v1.0.0 Beta
+
+This beta release focuses on **foundational stability** and a few
+carefully chosen architectural improvements:
+
+-   **Simpler Project Scaffolding**\
+    We've unified the starter templates into a single clean foundation.
+    Authentication is now available as an optional, version-locked
+    Passkey addon.
+
+-   **Improved Hydration and Streaming**\
+    Our server-side rendering pipeline has been re-architected to
+    eliminate `useId` hydration mismatches and improve streaming
+    performance.
+
+-   **UI Library Examples**\
+    We've added an end-to-end tested playground showcasing integration
+    with popular libraries including **shadcn/ui (Radix)**, **Base UI**,
+    and **Chakra UI**.
+
+-   **Consistent Middleware**\
+    RSC Actions now run through the same global middleware pipeline as
+    page requests, enabling consistent authentication, session, and
+    business logic across your entire app.
+
+------------------------------------------------------------------------
+
+## A Clear Roadmap to v1.0
+
+We want to be transparent about where we're headed:
+
+-   **1.0 Beta → Foundational Stability & Breaking Changes**\
+    Fixing show-stoppers and executing necessary breaking changes to
+    make the dev loop reliable.
+
+-   **1.0 → Public Surface Stability**\
+    Documentation improvements, clearer error messages, and solidifying
+    which features are stable vs. preview.
+
+-   **1.x → Growth & Hardening**\
+    Backwards-compatible features and continued polish on experimental
+    areas.
+
+-   **Future → Big Ideas**\
+    Larger quality-of-life improvements and architectural explorations.
```

Comment:
If you think it makes sense: we can also tell people we use labels on github to plan what's going on in each phase, e.g: https://github.com/redwoodjs/sdk/issues?q=is:issue%20state:open%20label:1.0

e.g. if they wanted visibility into what's going on when